versioned_sidebars/version-2.0.0-sidebars.json

```diff
@@ -87,15 +92,18 @@
             "quickstart/samples-bunjs",
             "quickstart/samples-nextjs",
             "quickstart/samples-nodejs",
-            "quickstart/samples-node-jwt"
+            "quickstart/samples-node-jwt",
+            "quickstart/express-mongodb"
```

Comment:
it will be expressjs-mongodb. please update it

Reply:
sure